### DISCUSSION TITLE
Support for larger diameters

### DISCUSSION THREAD

**Discussion by SeanldMcDnld** [VirtualPhotonics/MieSimulatorGUI#11] [General]
First off, thank you for this awesome piece of software!
I'm using it to generate a more accurate phase function for realtime rendering of clouds and rain. I don't own QT so I can't modify the source code myself, but it would be amazing if the mean diameter could be raised to more than 1000microns, and the poly disperse not capped either. Philip Laven's Mie Plot application isn't limited in this way, but it's very cumbersome to get data out and into Excel.

**Comment by janakarana** [comment]
The current version has an upper limit on mean diameter to maintain stability. We may consider increasing the upper limit in the future.
